docs/source/get_started/handle_text.rst

```diff
@@ -128,5 +128,5 @@ Then, :obj:`stype_encoder_dict` can be directly fed into
 :class:`~torch_frame.nn.encoder.StypeWiseFeatureEncoder` to handle text columns.
 
 Please refer to the
-`pytorch-frame/examples/ft_transformer_text.py <https://github.com/pyg-team/pytorch-frame/blob/master/examples/ft_transformer_text.py>`_
+`pytorch-frame/examples/transformers_text.py <https://github.com/pyg-team/pytorch-frame/blob/master/examples/transformers_text.py>`_
```

Comment:
Why did you rename it? the transformer here refers to ft_transfomer?

Reply:
I want to align the name with the [llm_embedding.py](https://github.com/pyg-team/pytorch-frame/blob/master/examples/llm_embedding.py)

Reply:
Got it. sounds good.

Reply:
Got it. sounds good.